k2/torch/csrc/fsa_class.cu

```diff
@@ -421,70 +427,61 @@ void RaggedArc::SetAttr(const std::string &name, py::object value) {
 
   all_attr_names.insert(name);
 
-  if (THPVariable_Check(value.ptr())) {
-    torch::Tensor tensor = value.cast<torch::Tensor>();
+  if (value.isTensor()) {
+    torch::Tensor tensor = value.toTensor();
     SetAttr(name, tensor);
     return;
   }
 
-  try {
-    RaggedAny ragged_tensor = value.cast<RaggedAny>();
-    SetAttr(name, ragged_tensor);
+  if (value.isCustomClass()) {
+    torch::intrusive_ptr<RaggedAnyHolder> ragged_any_holder =
```

Comment:
Please add a check that the value contained in the `IValue` is indeed an instance of `RaggedAny`.

Reply:
@csukuangfj  How to check it? Is there any documents.